### ISSUE TITLE
ssh auth from gpg key

### ISSUE THREAD

**Issue by nim65s** [zeapo/Android-Password-Store#334]
Hi, 

Open-Keychain recognizes that the gpg key on my yubikey has an authentification subkey.

Would it be possible for Android-Password-Store to use this gpg auth subkey to authenticate the ssh connexion for the git clone ?

This already works on my desktops, and would be really convenient on a smartphone too :)

**Comment by zeapo** [comment]
Hi,

This is a duplicate of #71 and is blocked by open-keychain/open-keychain#908

**Comment by nim65s** [comment]
thanks :)
